VAMobile/src/screens/HealthScreen/HealthScreen.tsx

```diff
@@ -41,9 +41,23 @@ export function HealthScreen({}: HealthScreenProps) {
   const navigateTo = useRouteNavigation()
   const { t } = useTranslation(NAMESPACE.COMMON)
   const isScreenContentAllowed = screenContentAllowed('WG_Health')
+  const isFocused = useIsFocused()
 
-  const unreadCount = useSelector<RootState, number>(getInboxUnreadCount)
   const { data: userAuthorizedServices } = useAuthorizedServices({ enabled: isScreenContentAllowed })
+  const smNotInDowntime = !useDowntime(DowntimeFeatureTypeConstants.secureMessaging)
+  const { data: foldersData, isFetched: smFetch } = useFolders({
+    enabled: isScreenContentAllowed && userAuthorizedServices?.secureMessaging && smNotInDowntime,
+  })
+  const [unreadCount, setUnreadCount] = useState(0)
+
+  useEffect(() => {
+    if (smFetch && foldersData && isFocused) {
```

Comment:
Instead of using `isFocused` in the effect, you can use it in the `enabled` option of the query. That way it won't trigger necessary re-renders, and it'll have the same effect

``` tsx
enabled: isFocused && isScreenContentAllowed && userAuthorizedServices?.secureMessaging && smNotInDowntime
```

The other instances of `isFocused` in the other files can be updated as well